app/src/main/java/com/example/tasky/itemdetail/domain/VisitorType.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.tasky.itemdetail.domain
+
+import com.example.tasky.R
+
+sealed class VisitorType(
+    val name: Int,
+    val isSelected: Boolean
+) {
```

Comment:
Since you can only have 1 VisitorType selected (Either All, Going or Not going)
Then having a isSelected boolean would require you to remember to set every other type to false when selecting a new one, and so on

One better solution could be:
1- Have VisitorType be an enum, and just have the name: Int attribute)
2- Have a "selectedFilter: VisitorType" or similar attribute on the state

And when clicking, just setting the filter to the selected one. So you will always have just 1 selected